_posts/2022-11-21-changelog-183.markdown

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #183"
+date: 2022-11-21 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#9694] Enable [derive](https://doc.rust-lang.org/reference/procedural-macros.html#derive-macros) procedural macro expansion in [nightly plugin builds](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-quick-start.html#install-nightly).
+          To turn it off, disable `org.rust.macros.proc.derive` [experimental feature](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-faq.html#experimental-features)
+
+* [#9641], [#9699] Support [Intention and quick-fixes preview](https://blog.jetbrains.com/idea/2022/09/intellij-idea-2022-3-eap/#Intention_action_preview_enabled_by_default) in 2022.3
```

Comment:
Maybe add a note that not all intention/quick-fixes have a preview at this moment and add a link to [meta](https://github.com/intellij-rust/intellij-rust/issues/9561) issue?